PyInstaller/loader/pyimod03_importers.py

```diff
@@ -298,8 +298,8 @@ def get_code(self, fullname):
             # next line will raise an execpion which will be catched just
             # below and raise the ImportError.
             return self._pyz_archive.extract(fullname)[1]
-        except:
-            raise ImportError('Loader FrozenImporter cannot handle module ' + fullname)
+        except BaseException as e:
```

Comment:
Please change this to `Exception`, not `BaseException`. 

```suggestion
        except Exception as e:
```